_inc/client/plans/plan-header.jsx

```diff
@@ -33,7 +33,7 @@ const PlanHeader = React.createClass( {
 									{ __( "Hackers, botnets and spammers attack websites indiscriminately. Their goal is to attack everywhere and often. Our goal is to help you prepare by blocking these threats, and in worst-case-scenarios we'll be here to help you restore your site to its former glory." ) }
 								</p>
 								<p className="jp-landing-plans__header-btn-container">
-									<Button href={ 'https://wordpress.com/plans/' + window.Initial_State.rawUrl } className="is-primary">
+									<Button href={ 'https://jetpack.com/redirect/?source=plans-main-top&site=' + window.Initial_State.rawUrl } className="is-primary">
```

Comment:
Let me handle this, I just build and tested it and it's missing that prop. Will commit to this branch.